single_kernel_mongo/config/literals.py

```diff
@@ -108,7 +108,7 @@ class VmUser(WorkloadUser[int]):
 
 MAX_PASSWORD_LENGTH = 4096
 
-PBM_RESTART_DELAY = 5
+RESTART_DELAY = 5
```

Comment:
why not `PBM_RESTART_DELAY `, like it was a few commits ago? I think thats more informative